src/cartesian/Area.tsx

```diff
@@ -66,8 +66,8 @@ interface AreaProps extends InternalAreaProps {
   tooltipType?: TooltipType;
   connectNulls?: boolean;
   // whether have dot in line
-  activeDot?: AreaDot;
-  dot?: AreaDot;
+  activeDot?: ActiveDotType;
```

Comment:
Should we not just have a dot type? dot isn't an "active" dot right?

Reply:
I am also confused. The original were identical so I used the same.

Reply:
lol. Are there differences between DotProps and ActiveDotType?

Reply:
`any` vs `any`